core/include/ldml/keyboardprocessor_ldml.ts

```diff
@@ -622,7 +622,7 @@ class Constants {
   /** minimum usable marker index */
   readonly marker_min_index    = 0x0001;
   /** index value referring to the 'any' marker match */
-  readonly marker_any_index    = 0xFFFE;
+  readonly marker_any_index    = 0xD7FF;
   /** maximum marker index prior to the 'any' value */
   readonly marker_max_index    = this.marker_any_index - 1;
   /** maximum count of markers (not including 'any') */
```

Comment:
next line should read `this.marker_max_index - this.marker_min_index + 1;`

Reply:
good catch! I also realized I had forgotten to put in the marker validation, which I said I would. That;'s now in.